bottlecap/src/logs/lambda/processor.rs

```diff
@@ -341,6 +341,8 @@ impl LambdaProcessor {
             for mut orphan_log in self.orphan_logs.drain(..) {
                 orphan_log.message.lambda.request_id =
                     Some(self.invocation_context.request_id.clone());
+                let should_send_log = self.logs_enabled
```

Comment:
Oh god, this failed because we shadowed the `should_send_log` variable? 

This reassignment works, but can we refactor this into an instance method which just borrows the message itself?

If that is too complicated to pass the borrow checker, can we instead rename this variable to `orphan_should_send_log` so we don't reshadow this?

Reply:
It's because we reused the result of `should_send_log` from the first message for all the log messages.
Will try to refactor this.